crates/evm/src/tests/tx_tests.rs

```diff
@@ -73,7 +72,7 @@ fn tx_conversion() {
         signer,
         signed_transaction: reth_primitives::TransactionSigned {
             hash: Default::default(),
-            signature: Default::default(),
+            signature: Signature::new(Default::default(), Default::default(), Default::default()),
```

Comment:
It is `SYSTEM_SIGNATURE`.

Reply:
cannot do a const system signature anymore because the Signature fields are now private and there is no const fn to create signature

Reply:
But I can do lazy_static

Reply:
> Just saw your pr, gotta wait for a release tho

I tried with the rev after your pr merged and it broke a lot of things, we really gotta wait for new reth release for that unfortunately, or a port pr